src/app/screens/Receive.tsx

```diff
@@ -232,16 +232,6 @@ function Receive() {
                   }
                 />
               </div>
-              <a
```

Comment:
How is this related to removing a shadow?

Reply:
I think that's a left over from: https://github.com/getAlby/lightning-browser-extension/pull/657